sql-plugin/src/main/spark350db143/scala/org/apache/spark/sql/rapids/execution/GpuSubqueryBroadcastMeta.scala

```diff
@@ -28,7 +28,7 @@ class GpuSubqueryBroadcastMeta(
     conf: RapidsConf,
     p: Option[RapidsMeta[_, _, _]],
     r: DataFromReplacementRule) extends
-    GpuSubqueryBroadcastMetaBase(s, conf, p, r) {
+    GpuSubqueryBroadcastMeta330DBBase(s, conf, p, r) {
```

Comment:
Why does Spark 4.0 inherent from the databricks base? Does that work? should if so should the base class be renamed? If not then we need to fix that bug.

Reply:
That's a good point. The 4.0 shim should probably remain where it was.

Reply:
I've sorted this out now.